### PR TITLE
Minorly Update Beatmaps

### PR DESCRIPTION
**Pull request by circleen**
Beatmaps is really outdated.

### CODE REVIEW

File: wiki/Beatmaps/en.md [hunk 1]
```diff
@@ -1,32 +1,28 @@
----
```
Comment (cl8n) [inline wiki/Beatmaps/en.md]: this PR doesn't do enough to remove this tag

**Comment by cl8n** [comment]
i cant tell when to review cuz uve been regularly updating for the last few days lol

**Comment by circleen** [comment]
probs review now

**Comment by circleen** [comment]
Closing because no activity since 11 days and unknown repository.

**Pull request closed without merging**